src/botPage/view/style.js

```diff
@@ -1,6 +1,11 @@
 export const content = {
     marginTop: '0.5em',
-    width    : '18em',
+};
+
+export const dialogLayout = {
+    height   : 'auto',
+    width    : 410,
```

Comment:
We usually set the sizes using `em` whenever possible, is there a reason for choosing pixels?

Reply:
You can use `"25em"` instead.